pkg/util/orchestrator/orchestrator.go

```diff
@@ -30,6 +30,9 @@ type Interface interface {
 
 	AuditRecovery(cluster string) ([]TopologyRecovery, error)
 	AckRecovery(id int64, commnet string) error
+
+	SetHostWritable(host string, port int) error
```

Comment:
You can use as parameter `InstanceKey` because instance key has both `host` and `port`